lib/CXGN/BrAPI/v2/ObservationUnits.pm

```diff
@@ -405,6 +405,9 @@ sub observationunits_update {
 
     foreach my $params (@$data) {
         my $observation_unit_db_id = $params->{observationUnitDbId} ? $params->{observationUnitDbId} : undef;
+        print "assign observation unit dbid\n";
+        print "$observation_unit_db_id";
+        print"\n";
```

Comment:
Did you want these print statements to stay in?  If so, I am cool with that.

Reply:
+1